src/kirby/testing/test-helper.ts

```diff
@@ -0,0 +1,26 @@
+export class TestHelper {
+  public static whenHydrated(node: Node, timeout: number = 2000) {
```

Comment:
When knowing the implementation that ionic components adds a "hydrated" class to shadow dom, you can understand what's going on here.... if I didn't have that knowledge... I would be thinking "what's going on here".

Consider adding a single line (or two) of TSDoc to the method?